deepchecks/nlp/checks/data_integrity/under_annotated_segments.py

```diff
@@ -250,16 +252,19 @@ class UnderAnnotatedPropertySegments(UnderAnnotatedSegments):
         number of segments with the weakest performance to show.
     categorical_aggregation_threshold : float , default: 0.05
         In each categorical column, categories with frequency below threshold will be merged into "Other" category.
+    multiple_segments_per_property : bool , default: False
+        If True, will allow multiple segments per feature otherwise will only allow one segment per property.
```

Comment:
```suggestion
        If True, will allow the same feature to be a segmenting feature in multiple segments, otherwise each feature can appear in one segment at most.
```